src/main/java/ch/njol/skript/expressions/ExprIndices.java

```diff
@@ -101,6 +102,12 @@ protected String[] get(Event e) {
 		if (sort) {
 			int direction = descending ? -1 : 1;
 			return variable.entrySet().stream()
+				.map((entry) -> new Pair<>(
+					entry.getKey(),
+					entry.getValue() instanceof Map<?,?>
+						? ((Map<?,?>) entry.getValue()).get(null)
+						: entry.getValue()
+				))
 				.sorted((a, b) -> ExprSortedList.compare(a.getValue(), b.getValue()) * direction)
 				.map(Entry::getKey)
```

Comment:
Not Entry anymore

Reply:
i was told to make it map to a pair by APickledWalrus, is that not what i should have done? Pair still has getKey and getValue and all tests pass

Reply:
It should be `.map(Pair::getKey)`, not `Entry::getKey`